src/Util.js

```diff
@@ -13,22 +13,15 @@ function commonParams() {
     };
 }
 
-const TagRegex = /(refs\/)(tags\/M-staged-PR)(\d+)$/;
+const PrNumberRegex = /(.* \(#)(\d+)(\)$)/;
```

Comment:
We certainly need to capture the PR number (i.e. the second set of unescaped parenthesis), but it is not clear why we need to capture the prefix string (i.e. why we need the first set of unescaped parenthesis). Remove as unneeded?

Same for the last set of unescaped parenthesis that capture the closing paren and the end-of-line.

Reply:
Done.